src/DssSpell.t.sol

```diff
@@ -904,63 +908,368 @@ contract DssSpellTest is DssSpellTestBase {
+        {
+            (uint256 srcInk, uint256 srcArt) = vat.urns(SRC_ILK, address(srcPsm));
+            assertEq(srcInk, psrcInk - expectedMoveWad, "after: src ink is not decreased by the moved amount");
+            assertEq(srcInk, srcKeep, "after: src ink is different from src keep");
+            assertEq(srcArt, psrcArt - expectedMoveWad, "after: src art is not decreased by the moved amount");
+            assertEq(vat.gem(SRC_ILK, address(srcPsm)), psrcVatGem, "after: unexpected src vat gem change");
+            assertEq(
+                _amtToWad(gem.balanceOf(address(srcPsm.gemJoin()))),
+                _amtToWad(psrcGemBalance) - expectedMoveWad,
+                "after: invalid gem balance for src pocket"
+            );
+        }
+
+        // Old PSM is properly configured on AutoLine
+        {
+            (uint256 maxLine, uint256 gap, uint48 ttl, uint256 last,) = autoLine.ilks(SRC_ILK);
+            assertEq(maxLine, srcMaxLine, "after: AutoLine invalid maxLine");
+            assertEq(gap, srcGap, "after: AutoLine invalid gap");
+            assertEq(ttl, srcTtl, "after: AutoLine invalid ttl");
+            assertEq(last, block.number, "after: AutoLine invalid last");
+        }
+
+        // New PSM state is set correctly
+        {
+            // LitePSM ink is never modified
+            (uint256 dstInk, uint256 dstArt) = vat.urns(DST_ILK, address(dstPsm));
+            assertEq(dstInk, pdstInk, "after: unexpected dst ink chagne");
+            // There might be extra `art` because of the calls to `fill`.
+            assertGe(dstArt, pdstArt + expectedMoveWad, "after: dst art is not increased at least by the moved amount");
+            assertEq(dai.balanceOf(address(dstPsm)), dstBuf, "after: invalid dst psm dai balance");
+            assertEq(vat.gem(DST_ILK, address(dstPsm)), pdstVatGem, "after: unexpected dst vat gem change");
+            assertEq(
+                _amtToWad(gem.balanceOf(address(pocket))),
+                _amtToWad(pdstGemBalance) + expectedMoveWad,
+                "after: invalid gem balance for dst pocket"
+            );
+        }
+
+        // New PSM is properly configured on AutoLine
+        {
+            (uint256 maxLine, uint256 gap, uint48 ttl, uint256 last, uint256 lastInc) = autoLine.ilks(DST_ILK);
+            assertEq(maxLine, dstMaxLine, "after: AutoLine invalid maxLine");
+            assertEq(gap, dstGap, "after: AutoLine invalid gap");
+            assertEq(ttl, uint48(dstTtl), "after: AutoLine invalid ttl");
+            assertEq(last, block.number, "after: AutoLine invalid last");
+            assertEq(lastInc, block.timestamp, "after: AutoLine invalid lastInc");
+        }
     }
 
-    // RWA tests
-    function testRWA001Update() public {
-        RwaLiquidationOracleLike oracle = RwaLiquidationOracleLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
-        GemAbstract rwa001 = GemAbstract(addr.addr("RWA001"));
+    function test_CRON_LITE_PSM_JOB() public {
```

Comment:
This test currently doesn't check that the old job is removed. Also nit: it doesn't sanity check that the new one isn't there yet.

Reply:
> This test currently doesn't check that the old job is removed.

Agreed with this.

> Also nit: it doesn't sanity check that the new one isn't there yet.
This is a redundant check, because the call would [fail](https://github.com/makerdao/dss-cron/blob/c779d94a13d61908c51ca17480701eeadd461743/src/Sequencer.sol#L120) if the same job was to be added 2x.

Reply:
> This is a redundant check, because the call would [fail](https://github.com/makerdao/dss-cron/blob/c779d94a13d61908c51ca17480701eeadd461743/src/Sequencer.sol#L120) if the same job was to be added 2x.

This still assumes that this job was indeed added in the spell, not skipped, not added to a some other sequencer.

Going forward, we might want to add a base test (similar to `testChainlogIntegrity`) to check cron jobs to avoid testing it manually.

Reply:
done in [ec331dd](https://github.com/makerdao/spells-mainnet/pull/422/commits/ec331dd901575e80df3e48f972fdd9e2859e5c0b)